src/main/java/by/itech/lab/supplier/service/impl/CustomerServiceImpl.java

```diff
@@ -69,6 +70,17 @@ public void delete(final Long id) {
     @Override
     @Transactional
     public void changeActive(Long id, boolean status) {
+        customerRepository.findById(id).map(elem -> {
+            elem.getUsers().forEach(u -> {
```

Comment:
Please, refactor it by extracting map and foreach functions to separate methods.